pom.xml

```diff
@@ -116,4 +116,7 @@
       <url>http://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+  <build>
+    <defaultGoal>clean install</defaultGoal>
```

Comment:
`install` is not something you should do. `verify` should be enough.

Reply:
done.